types/react-instantsearch-core/react-instantsearch-core-tests.tsx

```diff
@@ -335,75 +321,71 @@ import {
            */
           if ('items' in refinement) {
```

Comment:
Now that we made the other change I suggested (see here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43111/files#r392258447), Typescript believes that `items` can exist as a key in `refinement` and have a value of `undefined`.  That makes this the wrong check here.  The correct check is `if (refinement.items)`.  If you switch that, then Typescript will no longer complain about `refinement.items` being potentially undefined on line 324.